agent.py

```diff
@@ -35,14 +34,13 @@
 )
 from daemon import AgentSupervisor, Daemon
 from emitter import http_emitter
-from jmxfetch import JMXFetch, JMX_LIST_COMMANDS
 from util import (
     EC2,
     get_hostname,
-    get_os,
     Watchdog,
 )
 from utils.flare import configcheck, Flare
+from utils.jmx_command import jmx_command
```

Comment:
Hmm actually I get circular imports when the two are merged together, as `jmxfetch` imports `utils.jmx` (for the `JMXFiles` class) that itself imports `jmxfetch` (for the `JMXFetch` class used by `jmx_command`).

So unless there's a better solution I'll leave the two modules separated for now...